src/SFA.DAS.ApplyService.Application/Apply/Financial/StartFinancialReviewHandler.cs

```diff
@@ -1,29 +1,27 @@
 using System.Threading;
 using System.Threading.Tasks;
 using MediatR;
-using SFA.DAS.ApplyService.Domain.Entities;
+using Microsoft.Extensions.Logging;
 
 namespace SFA.DAS.ApplyService.Application.Apply.Financial
 {
-    public class StartFinancialReviewHandler : IRequestHandler<StartFinancialReviewRequest>
+    public class StartFinancialReviewHandler : IRequestHandler<StartFinancialReviewRequest, bool>
     {
         private readonly IApplyRepository _applyRepository;
+        private readonly ILogger<StartFinancialReviewHandler> _logger;
 
-        public StartFinancialReviewHandler(IApplyRepository applyRepository)
+        public StartFinancialReviewHandler(IApplyRepository applyRepository, ILogger<StartFinancialReviewHandler> logger)
         {
             _applyRepository = applyRepository;
+            _logger = logger;
         }
         
-        public async Task<Unit> Handle(StartFinancialReviewRequest request, CancellationToken cancellationToken)
+        public async Task<bool> Handle(StartFinancialReviewRequest request, CancellationToken cancellationToken)
         {
-            var section = await _applyRepository.GetSection(request.ApplicationId, 1, 3, null);
-
-            if (section.Status == ApplicationSectionStatus.Submitted)
-            {
-                await _applyRepository.StartFinancialReview(request.ApplicationId);   
-            }
-            
-            return Unit.Value;
+            _logger.LogInformation($"Starting financial review for application {request.ApplicationId}");
+            await _applyRepository.StartFinancialReview(request.ApplicationId, request.Reviewer);   
```

Comment:
You've removed the guard check... it was put in place to stop an application getting into a bad state due to an incorrect API call

Reply:
The SQL query prevents the state from being changed to an invalid value

Reply:
At present, there aren't section/sequence level statuses being set. The old code was retrieving the EPAO sequence/section from the database so was invalid for RoATP.

Reply:
Fair enough. Marking as resolved